src/backend/distributed/transaction/backend_data.c

```diff
@@ -710,6 +712,12 @@ AssignDistributedTransactionId(void)
 	pg_atomic_uint64 *transactionNumberSequence =
 		&backendManagementShmemData->nextTransactionNumber;
 
+	/* return idempotent transaction id for consecutive requests from a same session */
```

Comment:
Minor: update the function comment to say that this function is idempotent. (e.g., the last sentence of the comment now isn't accurate)